Makefile

```diff
@@ -42,14 +38,12 @@ _IMAGE = $(_DOCKER_REGISTRY)/$(_TEST_PROJECT)/$(DEV_IMAGE_NAME):$(DEV_IMAGE_TAG)
 _APB_IMAGE = $(_DOCKER_REGISTRY):5000/$(_TEST_PROJECT)/$(DEV_APB_IMAGE_NAME)
 _OPENSHIFT_USERNAME = developer
 _OPENSHIFT_PASSWORD = developer
-_TESTRUNNER_PORT = 80
+_DEV_IMAGE_STREAM = $(_IMAGE)
 else
 _DOCKER_REGISTRY = $(OPENSHIFT_ONLINE_REGISTRY)
 _IMAGE = $(_DOCKER_REGISTRY)/$(_TEST_PROJECT)/$(DEV_IMAGE_NAME):$(DEV_IMAGE_TAG)
-_TESTRUNNER_PORT = 80
-endif
-
 _DEV_IMAGE_STREAM = $(DEV_IMAGE_NAME):$(DEV_IMAGE_TAG)
```

Comment:
Hmm, the _DEV_IMAGE_STREAM is different in both branches above? Shouldn't this be just $(DEV_IMAGE_NAME):$(DEV_IMAGE_TAG) everywhere?

Reply:
I'm not entirely sure why but after the latest Docker updates on my local box, it requires me to put a full image name there (not just image stream name with a tag). Otherwise it results in generating this errors:

```
Failed to pull image "datagrid-services-dev:latest": rpc error: code = Unknown desc = repository docker.io/datagrid-services-dev not found: does not exist or no pull access
```

Of course, I could fix it by putting the OpenShift internal registry to my Docker configuration file, but I think we could use the full name here. This will allow other folks (like those with Macs) to use the same approach as we do.